week2/homework/src/methods.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs');
+let path = 'todo.txt';
+
+// CRUD Operation
+
+const method = {
+  add: buffer => {
+    fs.readFile(path, { encoding: 'utf-8' }, (err, data) => {
+      if (err) throw 'error writing file: ' + err;
+      buffer += data;
+      fs.writeFile(path, buffer, err => {
+        if (err) throw 'error writing file: ' + err;
+        console.log('wrote the file successfully');
+      });
+    });
+  },
+
+  list: () => {
+    fs.readFile(path, { encoding: 'utf-8' }, (err, data) => {
+      console.log(data);
+    });
+  },
+
+  remove: row => {
+    const removeLines = (data, lines = []) => {
```

Comment:
What happens if we try to remove an item with index 1000, or -1000? Consider the edge cases

Reply:
thank you 
I solved all issues